backend/homework/8/src/main/java/com/example/javahw8/controller/VehicleController.java

```diff
@@ -0,0 +1,54 @@
+package com.example.javahw8.controller;
+
+import com.example.javahw8.dto.VehicleDTO;
+import com.example.javahw8.entity.Vehicle;
+import com.example.javahw8.service.DTOMapper;
+import com.example.javahw8.service.VehicleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class VehicleController {
+
+    private VehicleService vehicleService;
+
+    @Autowired
+    public VehicleController(VehicleService vehicleService) {
+        this.vehicleService = vehicleService;
+    }
+
+    @PostMapping("/post")
+    public ResponseEntity<String> addVehicle(@RequestBody Vehicle vehicle){
+        vehicleService.addVehicleToInventory(DTOMapper.convertToDTO(vehicle));
+        return ResponseEntity.ok("Vehicle added successfully");
```

Comment:
Let the service function return ResponseEntity, Incase the function returns an exception the response can still be 200 ok